packages/forms/resources/views/components/tabs.blade.php

```diff
@@ -1,70 +1,47 @@
-<div
-    x-data="{
+<div x-data="{
```

Comment:
@bezhanSalleh, could you revert these formatting changes?

Reply:
yeah sure thing!